Lecture2/src/task4/model/MarathonStatistic.java

```diff
@@ -0,0 +1,18 @@
+package task4.model;
+
+public final class MarathonStatistic {
+
```

Comment:
private construnctor

Reply:
I have private constructor in this class 9 line.